extension/src/popup/helpers/sorobanSwap.ts

```diff
@@ -107,12 +107,27 @@ export const soroswapGetBestPath = async ({
 
           const data = await res.json();
 
+          return data;
+        },
+      },
+      {
+        protocol: Protocol.PHOENIX,
```

Comment:
apparently we needed to add another liquidity provider to ensure we could get a usable pathway for swap